lib/main.js

```diff
@@ -274,6 +288,16 @@ proto.rating = {
     if (el && !this.rating._isElSelected(el)) {
       el.click();
     }
+  },
+
+  // Reset the rating
+  resetRating: function () {
+    var selector = SELECTORS.rating.thumbSelectorFormat.replace('{rating}', this.rating.getRating());
+    var el = this.doc.querySelector(selector);
+
+    if (el) {
```

Comment:
I believe we are missing a `this.rating._isElSelected(el)` here -- to guarantee that we only uncheck a checked rating.